scripts/generator/graphql_generator/csharp/SDK/CartLineItems.cs.erb

```diff
@@ -391,14 +412,26 @@ namespace <%= namespace %>.SDK {
                     } catch(NoQueryException error) {}
                     #pragma warning restore 0168
 
-                    AddOrUpdate(checkoutVariantId, quantity, customAttributes);
+                    AddOrUpdate(checkoutVariantId, checkoutVariantPrice, quantity, customAttributes);
                     Get(checkoutVariantId).ID = checkoutLineItemId;
                 }
             }
 
             _IsSaved = true;
         }
 
+        private ProductVariant GetVariantById(Product product, string variantId) {
+            List<ProductVariant> variants = (List<ProductVariant>) product.variants();
+
+            foreach(var variant in variants) {
```

Comment:
Nit: I think you can use `Find` here instead of iterating over the list yourself:

```csharp
var variants = (List<ProductVariant>)product.variants();
return variants.Find(variant => variant == variantID);
```